### ISSUE TITLE
Simplified loss function

### ISSUE THREAD

**Issue by ShairozS** [asperti/BalancingVAE#2]
I'm porting this code over to Pytorch for use with a point cloud VAE - could you please provide a simplified/stand-alone version of the derived loss function? I'm especially having trouble with the calculation of the gamma, is the minimum taken between the previous batch MSE and current batch MSE?

**Comment by vincent-mayer** [comment]
I would also be interested, thanks!
